app/jobs/email_petition_signatories_job.rb

```diff
@@ -0,0 +1,80 @@
+class EmailPetitionSignatoriesJob < ActiveJob::Base
+  queue_as :default
+
+  def self.run_later_tonight(petition, requested_at, *extra_args)
+    set(wait_until: later_tonight).
+      perform_later(petition, requested_at.getutc.iso8601, *extra_args)
+  end
```

Comment:
We don't need to override this in all of the subclasses since we define a method `timestamp_name` anyway - also the `*extra_args` is never used as far as I can see. e.g.

``` ruby
def self.run_later_tonight(petition, requested_at = Time.current)
  petition.set_email_requested_at_for(timestamp_name, to: requested_at)
  perform_later(petition, requested_at.getutc.iso8601(6))
end
```

Note that I've changed the `iso8601` call to include a usec timestamp so we don't need a `to_i` comparison later